bin/xucli

```diff
@@ -17,6 +22,11 @@ require('yargs')
       describe: 'RPC service hostname',
       type: 'string',
     },
+    'rpc.tlscertpath': {
```

Comment:
I think we can drop the `rpc.` prefix, that's more for the config structure and here it's clear that the cli is interacting with our rpc api.

Reply:
We should drop it for all options (`port` and `host`) then. But I think we should keep it. It makes more clear what is meant and the user doesn't have to type more because they will be using the alias `-c` anyways

Reply:
Oh I had forgotten that we had `rpc.` for port and host. That was probably me that made it that way originally, ironically, but now I still think we don't need it for those options either. The `.` is a bit non-standard. I'd be ok with `rpcport`, `rpchost`, and `tlscertpath` options. That also seems more consistent with other services (bitcoind uses `rpcport` and `rpcconnect` while lnd uses `rpcserver` for both port and host).